public/less/take-a-stand.less

```diff
@@ -163,75 +192,104 @@
       }
     }
     .pol-pic-link {
-      font-size: 2rem;
+      // font-size: 2rem;
       font-weight: bold;
+      text-align: center;
       .text-font();
       > img {
         border-radius: 50%;
         border: 2px solid @blue;
         height: 18rem;
-        @media (min-width: @medium-bp) and (max-width: @large-bp) {
-          height: 15rem;
+        @media (max-width: @small-bp) {
+          width: 20rem;
+        }
+        @media (min-width: @small-bp) and (max-width: @medium-bp) {
+          height: 17rem;
+        }
+        @media (min-width: @medium-bp) {
+          height: 14rem;
         }
       }
     }
     .pol-name {
+      margin: auto;
+      text-align: center;
       margin-top: 1rem;
       min-height: 6rem; // uneven heights create irregular floating
       font-family: Oswald;
+      @media (max-width: @small-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @small-bp) and (max-width: @medium-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @medium-bp) {
+        font-size: 1.5rem;
+      }
     }
   }
 
   // ************************* STEP 4 *************************
   .step4 {
-    .twibbot-overlay-container {
-      // margin: auto;
-    }
-    .twibbot-overlay {
-      margin-bottom: 1em;
-      position: relative;
+    text-align: center;
+    .twibbon-overlay {
       margin: auto;
-      &, &:after, > img {
-        height: 100%;
-        width: 100%;
-        max-width: 40rem;
-      }
-      &:after {
-        content: '';
-        background: url('/photos/twibbot.png');
-        z-index: 0;
-        background-size: contain;
-        background-repeat: no-repeat;
-        position: absolute;
-        top: 0;
-        left: 0;
+      margin-bottom: 1em;
+      background: url('../photos/andy.jpg') no-repeat;
```

Comment:
The after created an overlay of the Stand with Duwamish Twibbon thing, so it was a picture of how Trish would look with the banner and it worked on all screen sizes but it was a little tricky so I can see how it would break with other changes. You could always combine the images in photoshop to avoid doing tricky stuff.